pom.xml

```diff
@@ -134,6 +134,31 @@
             </resource>
         </resources>
 
+        <pluginManagement>
```

Comment:
Why are you talking about `mvn site` if the question is about `checkstyle`? Style check is pretty quick. It is faster than compilation of the code. It makes sense to have it as a part of build process. Similar to unit tests.